src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,12 @@
+
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+
+:root {
+  --main-accent: #{$main-accent};
+  --secondary: #{$secondary};
+  --blue-accent: #{$blue-accent};
+  --white: #{$white};
+}
```

Comment:
we don't have to use `:root` in SCSS - just declare variables with `$` and use them